src/ape/api/networks.py

```diff
@@ -509,6 +509,18 @@ def enrich_calltree(self, call: CallTreeNode, **kwargs) -> CallTreeNode:
         """
         return call
 
+    @abstractmethod
+    def get_python_types(self, abi_type) -> List[Type]:
```

Comment:
```suggestion
    def get_python_types(self, abi_type: str) -> List[Type]:
```

Reply:
I think the right type here is `ABIType`

Reply:
it is potentially `Union[str, ABIType]`

Reply:
we can change `_python_type_for_abi_type` to also be `Union[str, ABIType]`, it should still work

Reply:
ok  just checked, youre right it is just ABIType i think